frontend/src/app/accountCreation/MfaSendCodeToContact/MfaSendCodeToContact.tsx

```diff
@@ -0,0 +1,135 @@
+import { useCallback, useEffect, useState } from "react";
+import { Redirect } from "react-router";
+
+import { Card } from "../../commonComponents/Card/Card";
+import { CardBackground } from "../../commonComponents/CardBackground/CardBackground";
+import TextInput from "../../commonComponents/TextInput";
+import Button from "../../commonComponents/Button/Button";
+import StepIndicator from "../../commonComponents/StepIndicator";
+import { accountCreationSteps } from "../../../config/constants";
+import { phoneNumberIsValid } from "../../patients/personSchema";
+import { emailIsValid } from "../../utils/email";
+import { capitalizeText } from "../../utils/text";
+
+interface Props {
+  type: "phone number" | "email address";
+  serviceEnroll: Function;
+  cardText: string;
+  cardHint?: string;
+}
+
+export const MfaSendCodeToContact = (props: Props) => {
+  const [contact, setContact] = useState("");
+  const [contactError, setContactError] = useState("");
+  const [formIsDirty, setFormIsDirty] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+
+  const contactIsValid =
+    props.type === "phone number" ? phoneNumberIsValid : emailIsValid;
+
+  const validateContact = useCallback(() => {
+    setFormIsDirty(true);
+    let error = "";
+    if (!contact) {
+      error = `Enter your ${props.type}`;
+      setContactError(error);
+      return false;
+    }
+
+    let valid;
+    try {
+      valid = contactIsValid(contact);
+    } catch (e) {
+      valid = false;
+    }
```

Comment:
Any reason why you do like that but not like this: 
```suggestion
    const valid = contactIsValid(contact);
```

Reply:
Yes, because I want to catch any exceptions thrown by `contactIsValid`, and just set valid to `false` in that case. If I accept your change, any exceptions thrown will crash the app

Reply:
I've looked at all functions that used here and didn't see any exceptions in any case you receive boolean value.

Reply:
That may be the case, but _just in case_ anything funky happens with phone or email validation, I would rather it be the case that the app doesn't crash, and instead valid gets set to false. I don't think it hurts anything to keep the try catch?